OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -0,0 +1,305 @@
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="68r" to="76v"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="77r" to="84v"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="85r" to="92v"/>
+                                            3, stub after 6
+                                            6, stub after 3
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="93r" to="101v"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q16" n="16">
+                                            <dim unit="leaf">7</dim>
+                                            <locus from="102r" to="108v"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q17" n="17">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="109r" to="114v"/>
+                                            stub between <locus target="#113"/> and <locus target="#114"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
+                                        <item xml:id="q18" n="18">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="115r" to="123v"/>
+                                            <note>A parchment guard has been placed around the quire.</note>
+                                        </item>
```

Comment:
The statement `<note>A parchment guard has been placed around the quire.</note>` has been removed in the latest version of this record because unnecessary. We should perhaps make a list of phenomena or statements which potentially generate redundancy and find a way to make them as much concise as possible. Among them, for instance, there is the old issue of the scribal phenomena "e1", "e2", etc. which might be effectively shortened in my view.